tools/image-generator/generator.py

```
@@ -0,0 +1,501 @@
+    def __init__(self, font_path="/home/webber/Setup/FiraCode Nerd Font Mono"):
+        self.font_path = font_path
+        self.colours = COLOURS
+        self.default_width = 1000
+        self.default_height = 600
+
+    def get_font(self, size, weight='regular'):
+        """Get Nerd Font with proper fallback"""
+        try:
+            font_file = os.path.join(self.font_path, FONTS['weights'][weight])
+            return ImageFont.truetype(font_file, size)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)